server/pub/permissions.js

```diff
@@ -18,6 +18,7 @@ export const getPermissions = async ({ userId, communityId, pubId, licenseSlug }
 		licenseSlug === 'cc-0';
 	const canManage = validLicenseSlug && scopeData.activePermissions.canManage;
 	const editProps = [
+		'doi',
```

Comment:
Only admins can update the DOI, so we need to add this to the list if `scopeData.activePermissions.canAdmin`.